community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -567,6 +567,10 @@
     public static final Setting<String> procedure_unrestricted =
             setting( "dbms.security.procedures.unrestricted", Settings.STRING, "" );
 
+    @Description("If dbms.killQueries give a verbose output, with information about which querys where not found." )
```

Comment:
This description could be better, maybe `Specifies whether or not dbms.killQueries produces a verbose output, with information about which queries were not found`?